src/features/hotspots/root/HotspotsView.tsx

```diff
@@ -124,6 +129,17 @@ const HotspotsView = ({
     setLinkedHotspotAddress(params?.address || '')
   }, [params])
 
+  useMount(() => {
+    if (startOnMap) {
+      setShowMap(true)
+      return
+    }
+
+    setTimeout(() => {
+      setShowMap(true)
```

Comment:
So the map loads after the sheet now?

Reply:
yeah, it seems better to me this way. It removes the awkward chop that happens when they load at the same time. Could you try it and see what you think? Probably best to do with a release build.

Reply:
ok, sweet.